src/ServicePulse.Host/vue/src/composables/serviceServiceControl.ts

```diff
@@ -67,56 +79,75 @@ export const newVersions = reactive({
   },
 });
 
-export const connections = reactive({
+interface ServiceControlInstanceConnection {
+  settings: { [key: string]: Object };
+  errors: string[];
+}
+
+interface MetricsConnectionDetails {
+  Enabled: boolean;
```

Comment:
should be camelCase

Reply:
The payload from the server is not camelCase, see https://particularsoftware.slack.com/archives/C368VHCTT/p1706660541812479

Reply:
could use https://www.npmjs.com/package/camelcase-keys on the API response (on all API responses to be consistent)